cmd/client/wallet/generated_wallet.ini.go

```diff
@@ -0,0 +1,44 @@
+package main
```

Comment:
do we need to check in the generated wallet.ini.go file?  Also, maybe we shall ignore it in .gitignore?

Reply:
I believe so, otherwise the travis check script will fail.